src/data/unicorns.json

```diff
@@ -121,4 +121,18 @@
     "color": "#7C4DFF",
     "roles": ["developer"]
   }
+  {
+    "id": "MDutro",
+    "name": "Micah Dutro",
+    "firstName": "Micah",
+    "lastName": "Dutro",
+    "description": "A non-profit lawyer turned budding web developer.",
+    "socials": {
+      "github": "MDutro",
+    },
+    "pronouns": "he",
+    "profileImg": "./Mdutro.jpg",
```

Comment:
Your image seems to be a little small. If you're able to find a higher-res version of the image, that would be great! Otherwise, your image might seem a bit fuzzy on parts of the site :)

Reply:
Hmmm, I should have a higher res version on my PC at home.